ComputeClient/Compute.Client/ApiUris.cs

```diff
@@ -18,6 +18,9 @@ public static class ApiUris
         /// <summary>	The MCP 2.10 prefix. </summary>
         public const string MCP2_10_PREFIX = "caas/2.10/";
```

Comment:
How come you didnt remove this? I would expect all functions to be bumped along to 2.11.

Reply:
Just updated the PR to use v2.11 of api